### ISSUE TITLE
Shifting to the centre of the disc

### ISSUE THREAD

**Issue by bjnorfolk** [dmentipl/plonk#35]
### Describe the bug

Hi, I'm attempting to shift to the disc centre for both snap images and an animation. In phantom I set a 5mj companion at 10au but the sink position in plonk is significantly far from this.

### To Reproduce
I rotate and shift the disc with:
```python
snap_file = 'hd100453_bec_run_5mj_10au/disc_00101.h5'
snap = plonk.load_snap(snap_file)
analysis.discs.rotate_face_on(snap)
com = analysis.total.center_of_mass(snap, sinks=False)
snap.translate(-com)
```
Then setting the units and printing the sink positions:
```python
snap.set_units(position='au')
print(snap.sinks['position'])
```
The output.
```python
[[1.0657612036467647 -0.7482025165997441 0.08040807544826903] [151.16832435718015 -17.357874795606797 -215.90554676449221] [-14942.404548791676 35342.100422533564 18413.824209907314]] astronomical_unit
```
If I print the mass with 
```python
print(snap.sinks['mass'])
```
The output:
```python
[3.3820768744152954e+30 3.9782982217579486e+29 9.490655967300002e+27] kilogram
```
Therefor the third sink is clearly the 5mj companion but the xyz coordinates are obviously not at a 10au radius.

Is this a plonk issue for a phantom issue?

As for the animation, I'm not sure how to centre the disc at all or label the sinks with a marker. Any tips?
I load the entire simulation with:
```python
sim = plonk.load_simulation(prefix='disc', directory='hd100453_bec_run_5mj_10au/')
```

### Environment (please complete the following information)

+ OS: macOS 11.2 
+ Python 3.8.8
+ Plonk version: 0.7.3

### Images
It appears that I'm shifting correctly to the star sink.
![image](https://user-images.githubusercontent.com/34231154/113088872-3dee9c80-9232-11eb-8fd0-bcb0bbedc281.png)

### Data
Here's the phantom disc.setup for reference
```
# input file for disc setup routine

# resolution
                  np =     1000000    ! number of gas particles

# units
           dist_unit =          au    ! distance unit (e.g. au,pc,kpc,0.1pc)
           mass_unit =      solarm    ! mass unit (e.g. solarm,jupiterm,earthm)

# central object(s)/potential
            icentral =           1    ! use sink particles or external potential (0=potential,1=sinks)
              nsinks =           2    ! number of sinks
             ibinary =           0    ! binary orbit (0=bound,1=unbound [flyby])

# options for binary
                  m1 =       1.700    ! primary mass
                  m2 =       0.200    ! secondary mass
            binary_a =        207.    ! binary semi-major axis
            binary_e =       0.320    ! binary eccentricity
            binary_i =         49.    ! i, inclination (deg)
            binary_O =         47.    ! Omega, PA of ascending node (deg)
            binary_w =         18.    ! w, argument of periapsis (deg)
            binary_f =        180.    ! f, initial true anomaly (deg,180=apastron)
               accr1 =       5.000    ! primary accretion radius
               accr2 =       5.000    ! secondary accretion radius

# options for multiple discs
      use_binarydisc =           F    ! setup circumbinary disc
     use_primarydisc =           T    ! setup circumprimary disc
   use_secondarydisc =           F    ! setup circumsecondary disc
      use_global_iso =           F    ! globally isothermal or Farris et al. (2014)

# options for circumprimary gas disc
      isetgasprimary =           0    ! how to set gas density profile (0=total disc mass,1=mass within annulus,2=surface density normalisation,3=surface density at reference rad\
ius,4=minimum Toomre Q)
    itapergasprimary =           F    ! exponentially taper the outer disc profile
   ismoothgasprimary =           F    ! smooth inner disc
        iwarpprimary =           F    ! warp disc
         R_inprimary =         20.    ! inner radius
        R_refprimary =         20.    ! reference radius
        R_outprimary =         60.    ! outer radius
       disc_mprimary =       0.003    ! disc mass
       pindexprimary =       1.000    ! p index
       qindexprimary =       0.250    ! q index
      posanglprimary =       183.5    ! position angle (deg)
         inclprimary =        14.9    ! inclination (deg)
          H_Rprimary =       0.050    ! H/R at R=R_ref
             alphaSS =       0.005    ! desired alphaSS

# set planets
          setplanets =           1    ! add planets? (0=no,1=yes)
            nplanets =           1    ! number of planets

# planet:1
            mplanet1 =       5.000    ! planet mass (in Jupiter mass)
            rplanet1 =         10.    ! planet distance from star
         inclplanet1 =        14.9    ! planet orbital inclination (deg)
         accrplanet1 =       0.250    ! planet accretion radius (in Hill radius)

# timestepping
             norbits =          10    ! maximum number of outer planet orbits
              deltat =       0.100    ! output interval as fraction of orbital period
```

**Comment by dmentipl** [comment]
Hi @bjnorfolk thanks for raising this issue.

I'm not sure exactly where the issue is. In your example there are three things going on, the rotation, the translation, and a change of units.

Could you try doing the rotation then check if that is good? Then, separately, do the translation and see if that is good. After that, you could share some data with me to take a closer look.

**Comment by bjnorfolk** [comment]
Hi @dmentipl, thanks for getting back to me. 

I've tried the translation/rotation separately with identical results. The translation shifts the sink positions only marginally given the low inclination. The rotation sets the star within 1AU of (x,y,z) = (0,0,0). However, ideally I'd want the star sink to be centred on (0,0,0).

It's concerning that the 5mj companion is no where near the 10AU position set in phantom. But I'm not sure if this is due to an incorrectly set disc.setup file or me incorrectly using plonk.

I've emailed you the h5 file.

Cheers, Brodie
